verification/src/http_server/routers/solidity.rs

```diff
@@ -31,8 +31,8 @@ impl Router for SolidityRouter {
             .app_data(self.cache.clone())
             .service(
                 web::scope("/verify")
-                    .route("/flatten", web::post().to(flatten::verify))
-                    .route("/standard_json", web::post().to(standard_json::verify)),
+                    .route("/multi_part/", web::post().to(multi_part::verify))
+                    .route("/standard_json/", web::post().to(standard_json::verify)),
```

Comment:
```suggestion
                    .route("/multi-part", web::post().to(multi_part::verify))
                    .route("/standard_json", web::post().to(standard_json::verify)),
```

Reply:
It is better to use hyphens instead of underscores in REST paths

And what is the reason to add slashes at the end?

Reply:
> maybe replace underscore in standard_json too?

I believe it would be better to do that in a separate PR (e.g. #46 which is related to making types and routes to correspond to the discussed Api)